maven-surefire-plugin/src/site/apt/examples/inclusion-exclusion.apt.vm

```diff
@@ -209,4 +209,41 @@ Inclusions and Exclusions of Tests
   Multiple formats can be additionally combined.
   This syntax can be used in parameters: <<<test>>>, <<<includes>>>, <<<excludes>>>, <<<includesFile>>>, <<<excludesFile>>>.
 
+* Tests from dependencies
+
+  By default, ${thisPlugin} will only scan for test classes to execute in the configured <<<testSourceDirectory>>>. To
```

Comment:
Here you should say that the feature VTC was introduced in version 2.21.0: `Since 2.21.0 ...`